validation/chOpenApiValidator.go

```diff
@@ -246,3 +234,33 @@ func handleParseError(pe *openapi3filter.ParseError) models.CHError {
 func findRoute(r routers.Router, req *http.Request) (route *routers.Route, pathParams map[string]string, err error) {
 	return r.FindRoute(req)
 }
+
+func (chv *CHValidatorImpl) GetSchema(ctx context.Context, openApiSpec, contextId string) error {
+	var err   error
+
+	callOnce.Do(func (){
+		chv.doc , err = loadSchemaFromFile(ctx, openApiSpec, contextId)
+		if err != nil {
+			log.ErrorC(contextId, err, log.Data{config.OpenApiSpecKey: openApiSpec, config.MessageKey: "unable to open Open API spec"})
+		} else {
+			if err = chv.doc.Validate(ctx); err != nil {
+				log.ErrorC(contextId, err, log.Data{config.MessageKey: "error occurred while trying to call kin-openAPI validation method"})
+			}
+		}
+	})
+
+	return err
+}
+
+func loadSchemaFromFile(ctx context.Context, openApiSpec, contextId string) (*openapi3.T, error){
+	loader := &openapi3.Loader{Context: ctx, IsExternalRefsAllowed: true}
+	abs, err := callFilepathAbs(openApiSpec)
+	if err != nil {
+		log.ErrorC(contextId, err, log.Data{config.MessageKey: "error occurred while retrieving absolute path of validation schema file"})
+		return nil, err
+	}
+	log.InfoC(contextId, fmt.Sprintf("Retrieved absolute path of validation schema "), log.Data{config.SchemaAbsolutePathKey: abs})
+
+	// Load the validation schema.
```

Comment:
Slightly outdated comment here, maybe Return schema?

Could also comment some other lines of code within this function, to make it clean what we're doing at each step?